tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -192,23 +192,6 @@ class WskBasicTests
             assetHelper.withCleaner(wsk.action, "twice") { (action, name) => action.create(name, defaultAction) }
     }
 
-    it should "create an action, then update its kind" in withAssetCleaner(wskprops) {
-        (wp, assetHelper) =>
-            val name = "createAndUpdate"
-            val file = Some(TestUtils.getTestActionFilename("hello.js"))
-
-            assetHelper.withCleaner(wsk.action, name) {
-                (action, _) => action.create(name, file, kind = Some("nodejs"))
-            }
-
-            // create action as nodejs (v0.12)
-            wsk.action.get(name).stdout should include regex (""""kind": "nodejs"""")
-
-            // update to nodejs:6
-            wsk.action.create(name, file, kind = Some("nodejs:6"), update = true)
-            wsk.action.get(name).stdout should include regex (""""kind": "nodejs:6"""")
-    }
-
```

Comment:
Do we have a cli test that updates a node action with a new kind that is say python?